example/spec/loadTest/runScaledTest.js

```diff
@@ -0,0 +1,362 @@
+/* eslint-disable no-await-in-loop */
+
+'use strict';
+
+const path = require('path');
+const fs = require('fs');
+const pWaitFor = require('p-wait-for');
+
+const { getAggregateMetricQuery, getInvocationCount } = require('@cumulus/integration-tests/metrics');
+const { generateIterableTestDirectories } = require('@cumulus/integration-tests/utils');
+const { LambdaStep } = require('@cumulus/integration-tests/sfnStep');
+const { listS3ObjectsV2 } = require('@cumulus/aws-client/S3');
+const { providers, collections, granules } = require('@cumulus/api-client');
+const { randomString } = require('@cumulus/common/test-utils');
+
+const {
+  buildAndExecuteWorkflow,
+  api: apiTestUtils,
```

Comment:
And sort here?

Reply:
Alpha sort all the things.  I wish this could be lintable.  Will fix.

Reply:
227f01512